_posts/2020-04-xx-introducing-arrow-c-data-interface.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: post
+title: "Introducing the Apache Arrow C data interface"
+description: "This post introduces the Arrow C data interface, a simple C-based
+interoperability standard to simplify interactions between independent users
+and implementors of the Arrow in-memory format."
+date: "2020-04-02 00:00:00 +0100"
+author: apitrou
+categories: [application]
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+Apache Arrow is a cross-language, platform-independent in-memory format
```

Comment:
"Apache Arrow **includes** a cross-language, platform-independent in-memory **columnar format**"

Reflecting expanding scope of the project. Still running into people who are surprised by the amount of software in the project. Therefore, we should be consistent about expressing things that the project "has" or "includes" versus what it "is" (to avoid some of these "project identity confusion" problems)